src/util/triplea/MapXMLCreator/MapXMLCreator.java

```diff
@@ -0,0 +1,839 @@
+						break;
+					case 10:
+						showInfoMessage(
+									"Here is a list of all the available unit attachments: movement, attack, defense, isAir, isSea, isAA, isFactory, canBlitz, isSub, canBombard, isStrategicBomber, isTwoHit, isDestroyer, isArtillery, isArtillerySupportable, isMarine, isInfantry, isParatroop, isMechanized, transportCapacity, transportCost, carrierCapacity, carrierCost",
+									"List Of Available Unit Attachment Choices (As Of TripleA 1.0.3.4)");
+						break;
+					case 11: // showInfoMessage("To change the production value of a territory, click on the territory and enter the territory's new production value in the window that appears.", "Help For Current Step");
+						break;
+					case 12: // showInfoMessage("To add a new canal, click on both of the land territories that form the canal(Touch the canal) and click yes when it asks for confirmation. To remove all the canals from the map, right click on one of the territories and click 'Yes' to confirm.", "Help For Current Step");
+						break;
+					case 13: // showInfoMessage("To change the initial owner of a territory, click on the territory and type in its new owner when it asks for it.", "Help For Current Step");
+						break;
+					case 14: // showInfoMessage("To change the units that a territory begins with, click on the territory and use the window that opens to set the territory's units. To do so, use the + and - buttons below each unit's name to change the quantity of each unit in the territory. You can also edit the text in the result textbox directly to change the territory units. Just type the name of each unit you want to add, followed by ':', followed by the unit quantitiy: Example: 'infantry: 1, artillery: 3, armour: 2, fighter: 1'.", "Help For Current Step");
+						break;
+					case 15:
+						final String[] settings = GameSettingsPanel.s_allSettings.split(", ");
+						Arrays.sort(settings);
+						StringBuilder sb = new StringBuilder();
+						sb.append("<html>Here is a list of all the available game settings:<table border=0><tr><td><table border=0>");
+						int setting_nr = 1;
+						char last_first_letter = 'z';
+						final int third_size_setting = (settings.length + 2) / 3;
+						for (final String setting : settings)
+						{
+							char first_letter = setting.charAt(0);
+							if (first_letter == last_first_letter)
+								sb.append("<tr><td>" + setting + "</td></tr>");
+							else
+								sb.append("<tr><td><big>" + first_letter + "</big>" + setting.substring(1) + "</td></tr>");
+							last_first_letter = first_letter;
+							if (setting_nr % third_size_setting == 0)
+								sb.append("</table></td><td><table border=0>");
+							++setting_nr;

```

Comment:
As a matter of style, prefer to use the settings_nr++, rather than "++". It's  more common to see the latter, the previous is generally reserved for the cases where you need an increment first.